ci/prepare.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: "Prepare the environment for running stf"
+  hosts: controller
+  tasks:
+    - name: "Update pip"
+      ansible.builtin.pip:
+        name: pip
+        state: latest
+        extra_args: "-U"
+
+    - name: "Install pre-reqs from pip"
+      ansible.builtin.pip:
+        name:
+            # https://stackoverflow.com/questions/64073422/importerror-cannot-import-name-oauth1session-from-requests-oauthlib
+          - requests==2.27.1
```

Comment:
These seem like something that should maybe live in `roles/stf-run-ci/requirements.txt`?

Reply:
Yes, they probably should. It's a minor improvement that would help with other systems using this role

Reply:
I added a requirements.txt here: https://github.com/infrawatch/service-telemetry-operator/pull/484 

The Depends-On in the PR description should check out the dependant PR and let both be checked together.
I'll push the update for this file shortly